welcome.css

```diff
@@ -74,4 +74,89 @@ html, body {
   text-align: center;
   font-size: 32px;
   color: olive;
-}
\ No newline at end of file
+}
+
+.fa-bars::before {
+  display: none;
+  cursor: pointer;
+  font-size: 22px;
+}
+
+/* serch how to desin navagition menu bar
+how to give responsive design to navagiton menu bar 
+virtical nav disign
+
+
+how to hide menu when click on icon 
+
+how to design three line icon when click icon, it show a menu 
+
+website nav menu bar for mobile screen 
+
+
+how to show menu on top of the page 
+
+*/
+ 
+
+input[type="checkbox"]{
+  display: none;
+}
+
+@media (max-width: 800px) {
+  .fa-bars:before{
+    display: block;
+  }
+
+  #click:checked ~ .menu-btn i:before{
+    content: "\f00d";
+  }
+
+  .menu-list {
+   position: fixed;
+   top: 80px;
+   /* left: -100%; */
+   height: 100vh;
+   width: 100%;
+   display: none;
+   text-align: center;
+   transition: all 0.3s ease;
+  }
+
+  .web-name-container {
```

Comment:
web name container ? what is that ?

Reply:
web name container is 
<img width="636" alt="Screenshot 2024-07-10 at 0 44 35 PM" src="https://github.com/scheme-dojo/scheme-dojo.github.io/assets/143992531/f54a3b0e-4392-42d0-9ed3-9ab81e94a9cc">
it contains name and version of the web, i well change it to the name that mre1gh san recommended.